packages/bulkapi/src/controllers/BulkApi.ts

```diff
@@ -178,26 +186,31 @@ export class BulkApiImpl implements BulkApi {
         const clientId = getStringParam(request, "clientId")
         const depthLimit = getIntegerParam(request, "depthLimit", Number.MAX_SAFE_INTEGER)
         const idList = request.body.ids
-        dbLogger.info("Api.getNodes: " + JSON.stringify(request.body) + " depth " + depthLimit + " clientId: " + clientId)
         if (isParameterError(depthLimit)) {
+            traceLogger.info("   * retrieve request: depthlimit error")
```

Comment:
I wonder if user errors should have an higher level of logging, such as warning

Reply:
That makes sense, and in this case I also think this might use the requestLogger instead of the traceLogger.